pyproject.toml

```
@@ -5,7 +5,7 @@ license = {file = "LICENSE"}
 readme = "README.rst"
 
 requires-python = "~=3.7"
```

Continuing from https://github.com/icesat2py/icepyx/pull/539#discussion_r1717852390, wanted to check if this `~=` pin was intentional? Just noticed it while updating the deps at https://github.com/conda-forge/icepyx-feedstock/pull/18

I know Python 4.0 won't be coming anytime soon, but an upper pin is a big pain in the packaging world (see https://prefix.dev/blog/the_python_packaging_debate), and I'd prefer to use `>=3.7` instead if that's ok

```suggestion
requires-python = ">=3.7"
```